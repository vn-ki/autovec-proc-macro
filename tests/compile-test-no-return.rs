use autovec::autovec;
#[autovec]
pub fn fn_2(x: usize) {

}

fn main() { }
