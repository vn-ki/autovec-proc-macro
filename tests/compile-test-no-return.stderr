error: custom attribute panicked
 --> $DIR/compile-test-no-return.rs:2:1
  |
2 | #[autovec]
  | ^^^^^^^^^^
  |
  = help: message: Function need to return a type
